core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -936,20 +936,27 @@ private FileInfo getFileInfoInternal(LockedInodePath inodePath)
    */
   private FileInfo getFileInfoInternal(LockedInodePath inodePath, Counter counter)
       throws FileDoesNotExistException, UnavailableException {
+    int inMemoryPercentage;
+    int inAlluxioPercentage;
     Inode inode = inodePath.getInode();
     AlluxioURI uri = inodePath.getUri();
     FileInfo fileInfo = inode.generateClientFileInfo(uri.toString());
     if (fileInfo.isFolder()) {
       fileInfo.setLength(inode.asDirectory().getChildCount());
     }
-    fileInfo.setInMemoryPercentage(getInMemoryPercentage(inode));
-    fileInfo.setInAlluxioPercentage(getInAlluxioPercentage(inode));
```

Comment:
Currently in Alluxio you can see the inAlluxio and inMemory percentage are 0 for a folder. When you don't set this for a folder, is the default still 0? Could you test the output from `alluxio fs stat <file/dir>` and put them in the description of this PR?

Reply:
<img width="511" alt="image" src="https://user-images.githubusercontent.com/17329931/160306409-72f6ae68-5ec0-4284-a59e-de57091b30b0.png">

FYI @xichen01

Reply:
For the directory
``` java    
    int inMemoryPercentage;
    int inAlluxioPercentage;
```
will not be set, their default value is 0

<img width="2788" alt="image" src="https://user-images.githubusercontent.com/32928346/160349343-9126e2de-64fb-4179-ac8d-41008a9c347c.png">

Reply:
Great thanks, could you put this image in the PR description so we all see